modules/update-service-overview.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='update-service-overview-{context}']
+== The {product-title} update service
+
+The {product-title} update service is the hosted service that provides over the air updates to both 
+{product-title} and RHCOS. It provides a graph, or diagram that contain
+_vertices_ and the _edges_ that connect them, of component Operators. The edges
+in the graph show which versions you can safely upgrade to. The cluster version
+Operator checks with the {product-title} update service and determines valid upgrades and upgrade paths
+based on current component versions and information in the graph. If you
+configure it to do so, the {product-title} update service sends the release artifacts that it needs to
+perform the upgrade to your image registry, and the cluster version Operator
+upgrades your cluster. By accepting automatic updates, you can automatically
+keep your cluster up to date with the most recent compatible components.
+
+To allow the {product-title} update service to provide only compatible updates, a release verification
+pipeline exists to drive automation. Each release artifact is verified for
+compatibility with supported cloud platforms and system architectures as well
+as other component packages. After the pipeline confirms the suitability of a 
+release, the {product-title} update service can apply the update to your cluster or notify you that it
+is available.
+
+The interaction between the registry and the {product-title} update service is different during
+bootstrap and continuous update modes. When you bootstrap the initial
```

Comment:
Do we want to talk about bootstrapping here?  It never hits the update service.  Or are their future plans to pull the initial update payload from the update service?

Reply:
I don't know about future plans. I'll comment this out.